docs/man/upsd.conf.txt

```diff
@@ -70,8 +70,12 @@ Multiple LISTEN addresses may be specified.  The default is to bind to
 compiled in).
 +
 To listen on all available interfaces, you may also use '0.0.0.0' for IPv4 and
-and '::' for IPv6.
-
+and '::' for IPv6. As a special case, `LISTEN * <port>` (with an asterisk)
+would try to listen on both IPv4 (`0.0.0.0`) and IPv6 (`::0`) wild-card IP
+addresses, subject to `upsd` command-line arguments or OS/kernel configuration.
+If the system supports "dual-stack" mode (IPv4-mapped IPv6) per RFC-3493
+then there would be one listening socket for both address types.
```

Comment:
same as above:
  - would
  - dual-stack
  - put v6 first because that's what it does